### PR TITLE
cmd/atlas: move cmd/atlas to its own module

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,18 +3,9 @@ module ariga.io/atlas
 go 1.17
 
 require (
-	entgo.io/ent v0.10.2-0.20220502113020-4ac82f5bb3f0
 	github.com/DATA-DOG/go-sqlmock v1.5.0
-	github.com/fatih/color v1.13.0
 	github.com/go-openapi/inflect v0.19.0
-	github.com/go-sql-driver/mysql v1.6.0
```
Comment (rotemtam) [inline go.mod:10]: 💯💯💯😍😍😍😍

**Comment by rotemtam** [comment]
Closes https://github.com/ariga/atlas/issues/684

**Review by rotemtam — COMMENTED** [review]
This means go run ariga.io/atlas/cmd/atlas won't work until we add those labels we discussed right ?

**Comment by a8m** [comment]
> This means go run ariga.io/atlas/cmd/atlas won't work until we add those labels we discussed right ?

Just a git tag. I plan to do it after merging.

**Comment by rotemtam** [comment]
> > This means go run ariga.io/atlas/cmd/atlas won't work until we add those labels we discussed right ?
> 
> Just a git tag. I plan to do it after merging.

Nice. Let’s discuss tomorrow how to integrate this with continuous push (canary)

File: cmd/atlas/go.mod [hunk 2]
```diff
@@ -0,0 +1,40 @@
+module ariga.io/atlas/cmd/atlas
```
Comment (masseelch) [inline cmd/atlas/go.mod:1]: I usually call this package `cli`, whatever you prefer though :heart:

File: go.mod (hunk 1 above)
Comment (masseelch) [reply go.mod:10]: AESOME!

File: internal/integration/cockroach_test.go [hunk 3]
```diff
@@ -78,12 +78,6 @@ func crdbRun(t *testing.T, fn func(*crdbTest)) {
 	}
 }
 
-func TestCockroach_EntRevisions(t *testing.T) {
-	crdbRun(t, func(t *crdbTest) {
-		testEntRevisions(t)
-	})
-}
-
```
Comment (masseelch) [inline internal/integration/cockroach_test.go:86]: Very crucial test. I don't feel well to remove them from master, even for a small time-frame.

Comment (masseelch) [reply internal/integration/cockroach_test.go:86]: We talked about it.

**Review by masseelch — APPROVED** [review]
See my comment about the package name please :blush:

**Pull request merged**